frontend/src/components/BurgerMenu/BurgerMenu.jsx

```diff
@@ -16,6 +17,7 @@ export default function BurgerMenu() {
   const closeSideBar = () => {
     setOpen(false);
   };
+  const navigate = useNavigate();
 
   return (
     <BurgerMenuContainer>
```

Comment:
Links serve this exact purpose, you know ?

Reply:
in a function?